Documentation/Books/Manual/Upgrading/Manually/Cluster.md

```diff
@@ -0,0 +1,129 @@
+--> TMP <---
+
+
+Upgrading a Cluster
+===================
+
+This page will guide you through the process of an in-place upgrade (see also
+[General Upgrade Information](../GeneralInfo/README.md)). The different
+nodes in a cluster can be updated one at a time without incurring downtime
+of the cluster and very short downtimes of the single nodes.
+
+
+Upgrading a Cluster on Debian & Ubuntu
+======================================
```

Comment:
Another H1 headline? If we make this a child of "Upgrading a cluster" H1 headline and cascade the children, then we end up with a lot of headline levels on this page (already 4 right now)